lib/lago_utils/lago_utils/license.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module LagoUtils
+  class License
+    def initialize(url)
+      @url = url
+      @premium = false
+    end
+
+    def verify
+      return if ENV['LAGO_LICENSE'].blank?
+
+      http_client = LagoHttpClient::Client.new("#{url}/verify/#{ENV['LAGO_LICENSE']}")
+      response = http_client.post({}, [])
```

Comment:
TBH I'm not sure why we don't use HTTP GET here since we just fetch some informations here

Reply:
that designed in the license service this way, but we can update it with a `GET`, wdyt @rsempe ?

Reply:
Yep, `GET` makes more sense to me as well

Reply:
let's go for a GET so!